src/sentry/static/sentry/app/views/settings/incidentRules/types.tsx

```diff
@@ -119,10 +120,25 @@ export type MetricActionTemplate = {
    */
   integrationId: number;
 
+  /**
+   * Name of the SentryApp. Like `integrationName`, this differentiates SentryApps from each other.
+   */
+  sentryAppName: string;
```

Comment:
@mgaeta I would not expect these to be required fields of the type

Reply:
👍